bootstrap/README.md

```diff
@@ -52,11 +59,23 @@ Input this value in ims_token in the Platform section of the config.yaml
        input_dataset_title: []
        file_replace_tenant_id: DSWRetailSalesForXDM0.9.9.9.json
        file_with_tenant_id: []
-       is_ouput_schema_different: True
+       is_ouput_schema_different: "True"
        output_mixin_title: []
        output_mixin_definition_title: []
        output_schema_title: []
        output_dataset_title: []
-
+        
 # How to run
 Navigate to the `bootstrap` directory and run `python bootstrap.py`
+
+# Output of the script
+1. If ingest_data is set to "True" and build_recipe_artifacts is also set to "True", the data ingestion workflow is 
+executed and the recipe artifacts are generated. The output is one of the following artifacts based on the kernel_type:
+    1. Spark generates a jar file
+    2. PySpark generates an egg file
+    3. Python and R would prompt for login and generate a source url
+These artifacts are then utilized to make a recipe.    
```

Comment:
This sentence, "These artifacts are then ..." is not formatted correctly.  Need a line break after the "Python and R" list item.
<img width="1041" alt="Screen Shot 2019-06-19 at 3 52 27 PM" src="https://user-images.githubusercontent.com/8116184/59806964-1a4fa180-92ab-11e9-8290-5bf97cd79d4d.png">